### DISCUSSION TITLE
Include file backup utility

### DISCUSSION THREAD

**Discussion by semioticrobotic** [elementary/triage#343] [Ideas]
Installing a basic file backup utility is one of the first things I do for new users I'm migrating to elementary. I'd love to have one built into OS8 by default.

**Comment by bigfatbird** [comment]
It would be so nice to have something on the level of Time Machine like macOS.

New computer? Install elementary and just plug in your hard drive to restore to the old files, apps and settings. Just like changing a pair of underwear, everything stays the same, just some new clothes

**Comment by wasantosfi** [comment]
Something like Timeshift app?
